nixos/modules/services/networking/gvpe.nix

```diff
@@ -46,12 +46,14 @@ in
 
       nodename = mkOption {
         default = null;
+        type = types.str;
```

Comment:
These either  need to become `lib.types`, or you can add `types` to the `inherit (lib) mkOption mkIf;` line above.